OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -849,6 +854,17 @@ private async Task<IEnumerable<string>> GetRegionAdminsIds(Address address)
         return regionAdminsIds;
     }
 
+    private async Task<IEnumerable<string>> GetAreaAdminsIds(Address address)
+    {
+        var areaAdminsIds = await areaAdminRepository
+            .GetByFilterNoTracking(a => a.CATOTTGId == address.CATOTTGId)
+            .Select(a => a.UserId)
+            .ToListAsync()
+            .ConfigureAwait(false);
+
+        return areaAdminsIds;
+    }
```

Comment:
(Performance) Please try to avoid excessive async/awaits

This method could be simplified to:
```
    private async Task<List<string>> GetAreaAdminsIds(Address address)
        => areaAdminRepository
            .GetByFilterNoTracking(a => a.CATOTTGId == address.CATOTTGId)
            .Select(a => a.UserId)
            .ToListAsync();
```
Please, consider same changes for other `Get...AdminsIds()`.